assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -274,7 +230,62 @@ const Cart = ( {
 								/>
 							)
 						) }
-						{ showShippingCosts && <ShippingCalculatorOptions /> }
+						{ showShippingCosts && (
+							<fieldset className="wc-block-cart__shipping-options-fieldset">
+								<legend className="screen-reader-text">
+									{ __(
+										'Choose the shipping method.',
+										'woo-gutenberg-products-block'
+									) }
+								</legend>
+								{ shippingRates ? (
+									<Packages
+										className="wc-block-cart__shipping-options"
+										renderOption={
+											renderShippingRatesControlOption
+										}
+										shippingRates={ shippingRates }
+										selected={ [
+											shippingRates[ 0 ]
+												.shipping_rates[ 0 ].rate_id,
+										] }
```

Comment:
Why isn't this using `selectedShippingRate`?

Reply:
That's only used in the editor with placeholder rates, that's why we don't pass an `onChange` prop (we don't want to allow changing it). That also means `selectedShippingRate` will not be set, so we pass the first id as the `selected`.

Reply:
I wonder if we should add an inline comment block to make that clear?

Reply:
Sounds good, added in 0537e729f6e193e2cfebbe42d979d60f6c9359cc. :+1: